docs/src/sde_fitting/neural_sde.md

```diff
@@ -85,12 +90,12 @@ Let's see what that looks like:
 
 ```julia
 # Get the prediction using the correct initial condition
-prediction0 = neuralsde(u0)
+prediction0, st1, st2 = neuralsde(u0,p1,p2,st1,st2)
 
-drift_(u, p, t) = drift_dudt(u, p[1:neuralsde.len])
-diffusion_(u, p, t) = diffusion_dudt(u, p[(neuralsde.len+1):end])
+drift_(u, p, t) = drift_dudt(u, p1, st1)[1]
+diffusion_(u, p, t) = diffusion_dudt(u, p2, st2)[1]
 
-prob_neuralsde = SDEProblem(drift_, diffusion_, u0,(0.0f0, 1.2f0), neuralsde.p)
+prob_neuralsde = SDEProblem(drift_, diffusion_, u0,(0.0f0, 1.2f0), [p1;p2])
```

Comment:
How should we handle passing the parameters of both chains here? Because doing `[p1,p2]` as above returns a flattened vector of the parameters which won't be restructured by `SDEProblem` I guess to call `drift_` and `diffusion_`

Reply:
Create a ComponentArray of the two ComponentArrays. They should nest fine, and then dereference them?